cmd/copyEnumeratorInit.go

```diff
@@ -255,6 +234,20 @@ func (cca *CookedCopyCmdArgs) initEnumerator(jobPartOrder common.CopyJobPartOrde
 			if cName == "" || (srcLevel == ELocationLevel.Service() && dstLevel > ELocationLevel.Service()) {
 				cName, err = containerResolver.ResolveName(object.ContainerName)
 
+				//attempt creating a container. We'll not create it multiple times.
+				err = cca.createDstContainer(cName, cca.Destination, ctx, existingContainers, azcopyLogVerbosity)
+
+				// if JobsAdmin is nil, we're probably in testing mode.
+				// As a result, container creation failures are expected as we don't give the SAS tokens adequate permissions.
+				// check against seenFailedContainers so we don't spam the job log with initialization failed errors
+				if _, ok := seenFailedContainers[cName]; err != nil && jobsAdmin.JobsAdmin != nil && !ok {
+					logDstContainerCreateFailureOnce.Do(func() {
+						glcm.Info("Failed to create one or more destination container(s). Your transfers may still succeed if the container already exists.")
+					})
+					jobsAdmin.JobsAdmin.LogToJobLog(fmt.Sprintf("failed to initialize destination container %s; the transfer will continue (but be wary it may fail): %s", bucketName, err), pipeline.LogWarning)
```

Comment:
(Unrelated to this PR)

We have a scanning log these days. Alternative to the log level change, we *could* move the location this is logged to to that, as customers tend not to look at that.